src/v2/mappers/ScheduleMapper.ts

```diff
@@ -12,4 +12,21 @@ export class ScheduleMapper {
       disciplineType: event.lessons[0].disciplineType.name,
     }));
   }
+
+  getEvent (event) {
+    return {
+      id: event.id,
+      name: event.name,
+      startTime: event.startTime,
+      endTime: event.endTime,
+      url: event.url,
+      description: event.description,
```

Comment:
description saved in eventInfo, so you need rewrite mapper. Also if event is pair, it will have several description, so you must send all of them or add week in query parameter